NEWS.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.10.13] - unreleased
+
+## Changed
```

Comment:
We probably need to note the rename of `PoincareBallTVector` to `PoincareBallTangentVector` ? And also for other such types? Do we keep the old names as deprecated aliases?

Reply:
Yes, right, I renamed all in a global search frenzy.

Since this is breaking I do not really want to keep them as deprecated.

Reply:
Ah, so this is going to be Manifolds 0.11?

Reply:
No!

You asked whether we shall keep the old names as deprecated – I wrote, _no_ I do not want that. We should just remove them since this is breaking.

Reply:
Why shouldn't it be 0.11 if we make breaking changes? I'm confused.

Reply:
Sure, we can hop back and forth for a while between 1.0 and 0.11 a few times now. For now all 3 PRs I am working on are prepared for 1.0. But if you want to go back to 0.11, sure. Then we go back. It was your idea to make this 1.0 in the first place,

Reply:
This is Manifolds.jl, not ManifoldsBase.jl. I'm talking about Manifolds.jl here.

Reply:
Sure, then we should add deprecations here as well. I also think, collecting deprecations is the better idea.

Reply:
I added the deprecations as a point to resolve here still.